monkey/common/di_container.py

```diff
@@ -111,13 +164,23 @@ def _retrieve_registered_instance(self, arg_type: Type[T]) -> T:
 
     def release(self, interface: Type[T]):
         """
-        Deregister's an interface
+        Deregister an interface
 
         :param interface: The interface to release
         """
         DIContainer._del_key(self._type_registry, interface)
         DIContainer._del_key(self._instance_registry, interface)
 
+    def release_convention(self, type_: Type[T], name: str):
+        """
+        Deregister a convention
+
+        :param **type_**: The `type` (class) of the dependency
```

Comment:
Why is it enclosed in `**`?

Reply:
Off-topic, but if we're fixing docstrings we should fix incorrect `MutableMapping` in `_del_key`

Reply:
> Why is it enclosed in `**`?

@ilija-lazoroski will remember. Something to do with a sphinx warning.

> Off-topic, but if we're fixing docstrings we should fix incorrect `MutableMapping` in `_del_key`

Fixed!

Reply:
There is some kind of problem when we have `_` before or after param's name, so we enclose it with `** **` to bold it out and fix that problem. We will probably encounter some of those things when we actually create the documentation.